src/SignalSlot/RemoveTranslationSlot.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\SignalSlot;
+
+use EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface;
+use eZ\Publish\Core\SignalSlot\Signal;
+use eZ\Publish\SPI\Persistence\Handler as PersistenceHandler;
+
+class RemoveTranslationSlot extends AbstractContentSlot
+{
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Handler
+     */
+    protected $persistenceHandler;
+
+    /**
+     * @param \EzSystems\PlatformHttpCacheBundle\PurgeClient\PurgeClientInterface $purgeClient
+     * @param \eZ\Publish\SPI\Persistence\Handler $persistenceHandler
+     */
+    public function __construct(
+        PurgeClientInterface $purgeClient,
+        PersistenceHandler $persistenceHandler
```

Comment:
`RemoveTranslationSignal` does not carry information about locations, so I had do make dependency on `PersistenceHandler`. Not sure if this is the optimal way, though. Any POV?
Just in case pushed whole location fetching as a fixup: 21caa86 so can be easily removed.